### ISSUE TITLE
Move .basilisk_ii_prefs and .basilisk_ii_xpram into ~/.var/app/

### ISSUE THREAD

**Issue by ssokolow** [flathub/net.cebix.basilisk#1]
As a general matter of principle, it's a good idea to play nicely with `flatpak override` or Flatseal so users can tighten the sandbox easily.

Setting `HOME` to `~/.var/app/net.cebix.basilisk` appears to do the trick with no ill effects if you don't want to patch the source.

**Comment by hfiguiere** [comment]
or submitting a patch upstream to use xdg-directories

**Comment by ssokolow** [comment]
True.

I took it for granted that was the best solution, so I didn't think to mention it when I suggested `HOME` as an option requiring less time and/or skill.

**Comment by SegHaxx** [comment]
Yeah using XDG base directories properly is something on the list of Things that Should Be Done, just hasn't been a high priority compared to some other things.

https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html

**Comment by SegHaxx** [comment]
Okay heads up, spent the day implementing XDG base directories. It will be making its way into the next release.

**Comment by SegHaxx** [comment]
Alright new build is on its way ea4fdbfd00c8affb7152bd293cf61106ec1fa467
